sdk/metricsadvisor/azure-ai-metricsadvisor/README.md

```diff
@@ -320,20 +317,20 @@ final AnomalyAlertConfiguration anomalyAlertConfiguration
 ```
 #### Query anomaly detection results
 This example demonstrates how a user can query alerts triggered for an anomaly detection configuration and get anomalies for that anomalyAlert.
-
+<!-- embedme ./src/samples/java/com/azure/ai/metricsadvisor/ReadmeSamples.java#L256-L276 -->
 ```java
-String alertConfigurationId = "9ol48er30-6e6e-4391-b78f-b00dfee1e6f5";
+long alertConfigurationId = "9ol48er30-6e6e-4391-b78f-b00dfee1e6f5";
```

Comment:
Should the value type here still be String?

Reply:
thanks, corrected